test/broadcast.jl

```diff
@@ -337,3 +337,9 @@ end
 @test broadcast(+, 1.0, (0, -2.0)) == (1.0,-1.0)
 @test broadcast(+, 1.0, (0, -2.0), [1]) == [2.0, 0.0]
 @test broadcast(*, ["Hello"], ", ", ["World"], "!") == ["Hello, World!"]
+
+# Issue #18379
+@test (+).(1, Ref(2)) == fill(3)
```

Comment:
cute, I didn't realize that `fill` could be used to create a 0-dimensional array like this